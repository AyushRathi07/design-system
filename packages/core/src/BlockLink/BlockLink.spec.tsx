import React from 'react'
import rendererCreateWithTheme from '../../test/rendererCreateWithTheme'

import { BlockLink } from '..'

describe('BlockLink', () => {
  test('renders', () => {
    const json = rendererCreateWithTheme(
      <BlockLink>raw text</BlockLink>
    ).toJSON()
    expect(json).toMatchSnapshot()
  })
})
