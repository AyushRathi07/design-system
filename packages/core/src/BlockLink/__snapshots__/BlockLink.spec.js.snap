// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlockLink renders 1`] = `
.c0 {
  display: block;
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
}

<a
  className="sc-qXTOB c0"
>
  raw text
</a>
`;
